shoop/admin/modules/system/views/telemetry.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+
+from django.http.response import HttpResponseRedirect, HttpResponse
+from django.views.generic.base import TemplateView
+from shoop.core.telemetry import is_opt_out, get_last_submission_time, get_telemetry_data, set_opt_out, \
```

Comment:
Don't use backslashes (`\`)... I would consider importing this as `from shoop.core import telemetry`.